README.md

```
@@ -2,68 +2,48 @@
+**Undo/Redo**: Full action history with a panel to jump to any prior state of the resolver.
 
-`nbdime` is a valuable notebook diff/merge tool in the Jupyter ecosystem.
+**Syntax Highlighting**: [CodeMirror](https://codemirror.net/)-powered highlighting for Python, Scala, R, Julia, and any other Jupyter-supported language.
 
-That being said, MergeNB focuses on a different experience:
 
-### Philosophy
+## Documentation
 
-Keep notebook conflict resolution as simple as possible inside an existing Git workflow.
+**Find the docs site [here](https://avni2000.github.io/MergeNB/docs)**
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)